src/utils/jump_utils.jl

```diff
@@ -66,16 +66,27 @@ function to_array(array::DenseAxisArray{<:Number})
 end
 
 function to_array(array::SparseAxisArray)
```

Comment:
@daniel-thom there are some coming changes to this function to support Hybrid Systems